doc/ansible/dev/developer_guide.md

```diff
@@ -390,6 +390,22 @@ can be used as shown:
       foo: bar
 ```
 
+### Ansible Operator Conditions
+The Ansible Operator has a set of conditions which it will use as it performs
+its reconciliation procedure. There are only a few main conditions:
+
+* Running - the Ansible Operator has called Run on the Ansible Runner to invoke
```

Comment:
`the Ansible Operator is currently running the ansible for reconcilation` WDYT? 

Ansible Runner to invoke your Ansible logic seems like it is too much about the details? Mostly devil's advocate but just asking.